4/Dockerfile

```diff
@@ -42,15 +42,17 @@ COPY	*.php /var/www/html/
 
 ENV	ADMINER_VERSION 4.8.1
 ENV	ADMINER_DOWNLOAD_SHA256 2fd7e6d8f987b243ab1839249551f62adce19704c47d3d0c8dd9e57ea5b9c6b3
-ENV	ADMINER_SRC_DOWNLOAD_SHA256 ef832414296d11eed33e9d85fff3fb316c63f13f05fceb4a961cbe4cb2ae8712
+ENV	ADMINER_COMMIT 1f173e18bdf0be29182e0d67989df56eadea4754
 
 RUN	set -x \
-&&	curl -fsSL https://github.com/vrana/adminer/releases/download/v$ADMINER_VERSION/adminer-$ADMINER_VERSION.php -o adminer.php \
+&&	apk add --no-cache --virtual .build-deps git \
+&&	curl -fsSL "https://github.com/vrana/adminer/releases/download/v$ADMINER_VERSION/adminer-$ADMINER_VERSION.php" -o adminer.php \
```

Comment:
Can't you just run https://github.com/vrana/adminer/blob/master/compile.php after cloning the git repo including the submodules? You won't have to download this release and can just compile it from the already locally existing sources.

Reply:
I probably can, but I preferred to use the “official build” for the actual Adminer “binary”.